src/main/java/axoloti/PlatformNull.java

```diff
@@ -0,0 +1,254 @@
+/**
+ * Copyright (C) 2013 - 2016 Johannes Taelman
+ *
+ * This file is part of Axoloti.
+ *
+ * Axoloti is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Axoloti is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * Axoloti. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Mark Harris
+ */
+package axoloti;
+
+import axoloti.attributedefinition.AxoAttributeComboBox;
+import axoloti.inlets.InletBool32;
+import axoloti.inlets.InletCharPtr32;
+import axoloti.inlets.InletFrac32;
+import axoloti.inlets.InletFrac32Buffer;
+import axoloti.inlets.InletInt32;
+import axoloti.object.AxoObject;
+import axoloti.object.AxoObjectInstanceAbstract;
+import axoloti.outlets.OutletBool32;
+import axoloti.outlets.OutletCharPtr32;
+import axoloti.outlets.OutletFrac32;
+import axoloti.outlets.OutletFrac32Buffer;
+import axoloti.outlets.OutletInt32;
+import axoloti.parameters.ParameterInstance;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class PlatformNull extends PlatformBase implements IPlatform {
```

Comment:
What's the use case for PlatformNull? Additionally, what are examples of other platforms that this feature would support in the future?